localstack/services/acm/provider.py

```diff
@@ -1,22 +1,42 @@
+from moto import settings as moto_settings
 from moto.acm import models as acm_models
 
 from localstack.aws.api import RequestContext, handler
-from localstack.aws.api.acm import AcmApi, RequestCertificateRequest, RequestCertificateResponse
+from localstack.aws.api.acm import (
+    AcmApi,
+    ListCertificatesRequest,
+    ListCertificatesResponse,
+    RequestCertificateRequest,
+    RequestCertificateResponse,
+)
 from localstack.services import moto
 from localstack.utils.objects import singleton_factory
 from localstack.utils.patch import patch
 
+# reduce the validation wait time from 60 (default) to 10 seconds
+moto_settings.ACM_VALIDATION_WAIT = min(10, moto_settings.ACM_VALIDATION_WAIT)
+
 
 @patch(acm_models.CertBundle.describe)
 def describe(describe_orig, self):
-    result = describe_orig(self)
+
+    # Terrible hack (for parity). Moto adds certain required fields only if status is PENDING_VALIDATION. TODO fix!
```

Comment:
Maybe we could add the TODO to the beginning of the sentence to enable IDE highlighting?

And this basically means that we should do an upstream fix to also add these mandatory fields even if the current status is not `PENDING_VALIDATION`, in the following section:
https://github.com/localstack/moto/blob/576c9122ff9b25548d0dca0cf22d737ef681b3ff/moto/acm/models.py#L343-L364

In fact, it seems like we could provide the whole patch to upstream (since it does not depend on any `localstack-ext` internals), however, this is a difficult question (when it comes to the coupling, and the reasoning for ACM in ext).

Reply:
Moved the TODO comment 👍  

Good point regarding the upstreaming. In fact, I'd love to discuss with you a migration of the entire logic into LocalStack. The moto ACM core implementation (models.py) has ~550 lines of code (which is not a lot), and our own provider is already at ~180 LOC. Seems like this could be a good case for a migration/re-implementation, and then we can add more proper snapshot tests and get rid of the patches.. 🚀